test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
+use Math;
+
+/** TestMatrix tests the functionality of the Jama Matrix class and associated decompositions.
+<P>
+Run the test from the command line using
+<BLOCKQUOTE><PRE><CODE>
+ java Jama.test.TestMatrix 
```

Comment:
@lydia-duncan yep. just went through trying to remove some of those artifacts - thanks for double checking the testing code that was ported over!